src/nav/new-relic-solutions.yml

```diff
@@ -1,11 +1,11 @@
 title: Guides and best practices
 path: /docs/new-relic-solutions
 pages:
-  - title: Observability Maturity
+  - title: Observability maturity
     pages:
       - title: Introduction
         path: /docs/new-relic-solutions/observability-maturity
-      - title: Uptime, Performance & Reliability
+      - title: Uptime, performance & reliability
```

Comment:
I wonder if we need to change the ampersand to "and"?

Reply:
In Gatsby build, the "Observability maturity" dropdown works just fine!. But When I click "Introduction" it goes to the former landing page just fine but that doc title still shows "Observability Maturity" capital M. Intentional? https://docswebsitedevelop-rhsfixobsmaturityconflict.gtsb.io/docs/new-relic-solutions/observability-maturity/introduction/  
I also checked all the dropdowns and links from this page. T and they go to the right docs. Thanks for taking care of this!

Reply:
@barbnewrelic, I changed that heading to "Observability maturity" to follow our style guide.